environment.yml

```diff
@@ -0,0 +1,17 @@
+name: odg
+channels:
+  - conda-forge
+dependencies:
+  - joblib
+  - shapely
+  - intake
+  - pandas
+  - xarray
+  - requests
+  - flake8
+  - isort
+  - erddapy
+  - pytest
+  - ipython
+  - aiohttp
+  - intake-xarray
```

Comment:
Also need https://github.com/intake/intake-xarray

Reply:
It's the last dependency listed.

Reply:
duh sorry that I missed that!